Clarify OpenCADD installation (issue #127) (#132)
volkamerlab/opencadd: docs/installing.rst

1. Create a new conda environment called ``opencadd``::

>     mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/user_env.yaml -n opencadd-dev

Reviewer: ```suggestion
    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/user_env.yaml 
```
Since `user_env.yaml` is the file excluding packages for testing and documentation, we can skip the `-n` here. The new env will be assigned accordingly to what is specified in `user_env.yml`.

>     # Or for contributors:
>     # We recommend to install the test environment, which includes all necessary packages for testing and documentation
>     mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/test_env.yaml -n opencadd-dev

Reviewer: Here I see a potential source of confusion because we are assigning the same name to two different environments (see line 36). What I suggest is to create a new subsection at the end that you can call "Development version" and then you add this part.
```suggestion
To install the development version of OpenCADD, you can run::
    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/test_env.yaml -n opencadd-dev
```
And then instructions to activate and how to run tests if needed, etc. IMO, showing a separate subsection for dev version is very helpful for the reader.

Author: Wonderful, I like this a lot! Please check if this is what you had in mind:
https://opencadd.readthedocs.io/en/joss-review-issue-127/installing.html#development-version

Reviewer: Looks very cool. Thanks for adding this!
